web-registry/src/pages/Dashboard/MyBridge/MyBridge.tsx

```diff
@@ -1,32 +1,43 @@
-import { Box } from 'web-components/lib/components/box';
+import { tabsStyles } from 'styles/tabs';
+
+import { Flex } from 'web-components/lib/components/box';
 import Card from 'web-components/lib/components/cards/Card';
 import { IconTabProps } from 'web-components/lib/components/tabs/IconTab';
 import { IconTabs } from 'web-components/lib/components/tabs/IconTabs';
 
-import { BridgableEcocreditsTable } from 'components/organisms/BridgableEcocreditsTable/BridgableEcocreditsTable';
-import { BridgedEcocreditsTable } from 'components/organisms/BridgedEcocreditsTable/BridgedEcocreditsTable';
-
-import { BridgeInfo } from './MyBridge.BridgeInfo';
-import { cardStyles, tabsStyles, wrapperStyles } from './MyBridge.styles';
+import { useLedger } from 'ledger';
 
-const tabs: IconTabProps[] = [
-  {
-    label: 'Bridgable ecocredits',
-    content: <BridgableEcocreditsTable />,
-  },
-  {
-    label: 'Bridged ecocredits',
-    content: <BridgedEcocreditsTable />,
-  },
-];
+import { BridgeInfo } from 'components/molecules';
+import {
+  BridgableEcocreditsTable,
+  BridgedEcocreditsTable,
+} from 'components/organisms';
 
 export const MyBridge = (): JSX.Element => {
+  const { wallet } = useLedger();
+
+  const tabs: IconTabProps[] = [
```

Comment:
Should we have a `useMemo` here to prevent rerendering for IconTabs?

Reply:
I guess that's better, thanks!